redundant-bmc/src/active_role_handler.cpp

```diff
@@ -92,7 +94,7 @@ sdbusplus::async::task<> ActiveRoleHandler::siblingHBStarted()
         sibling.waitForSiblingRole(), sibling.waitForBMCSteadyState());
 
     lg2::info("Attempting to enable redundancy now that sibling is back");
-    redMgr.determineAndSetRedundancy();
+    co_await redMgr.determineRedundancyAndSync();
```

Comment:
If the Active BMC reboots, there’s a possibility that PELs might have been created, deleted, or updated during that time in the Passive BMC. So, we’ll need to perform a full sync from Passive to Active in such cases as well.

Are you planning to handle that in a separate PR?